doc/METADATA.ini

```diff
@@ -614,3 +614,9 @@ usedby/plugins= crypto
 type= string
 description= used to mark a key for encryption. The crypto plugin is supposed to encrypt the key. After the encryption the type of the key will be set to "binary". Its original type will be restored after decryption.
 
+[crypto/salt]
+status= implemented
+usedby/plugins= crypto
+type= string
+description= used to store the salt thas is required for deriving the cryptographic key as well as the initialization vector (IV).
```

Comment:
Can you describe the life-time of the salt? Is it a security problem if the salt leaks?

I think its best if you add some paragraphs about "security considerations" in the docu somewhere.